internal/pkg/deploy/upload/customresource/customresource.go

```diff
@@ -84,15 +83,15 @@ func (cr *CustomResource) init() error {
 	for _, file := range cr.files {
 		f, err := w.Create(file.name)
 		if err != nil {
-			return fmt.Errorf("create zip file %q for custom resource %q: %v", file.name, cr.FunctionName(), err)
+			return fmt.Errorf("create zip file %q for custom resource %q: %v", file.name, cr.name, err)
```

Comment:
nit: can we use `.Name()` instead to rely on the public interface of the type rather than the internal data structure

Reply:
> Should we update the TemplateVersion: 'v1.1.0' field in the app template and deploy.LatestAppTemplateVersion to v1.2.0 ?

No we don't because it is not released yet.